ara/templates/playbook_file_summary.html

```diff
@@ -0,0 +1,26 @@
+{% extends "layout.html" %}
```

Comment:
the fact that a file is primary (or playbook) should probably be highlighted somehow